### DISCUSSION TITLE
Collapsible Paragraphs

### DISCUSSION THREAD

**Discussion by nicolovitelli** [squidfunk/mkdocs-material#7476] [Q&A] [answered]
Hi, I'd like to know if there is a way and how to implement collapsible paragraphs in the left section:

![image](https://github.com/user-attachments/assets/b9f533da-350b-47b4-b6bb-700c23db5065)

For example, in my documentation I'd like to make the "DDL Statements" collapsible, so when you click on it, it hides all the paragraphs.
Thank you for the help!

**Comment by squidfunk** [comment, marked as answer]
Currently not possible and not planned, as this is the table of contents, and we hear this feature request for the very first time. It can likely be done with some JS/CSS knowledge, but we're currently not considering this change.
